### PR TITLE
Desktop layout implemented

### PR DESCRIPTION
**Pull request by Aqurds**
Desktop version build by following,

- Media query
- Flex layout
- Width in percentage
- Images resized

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -409,3 +428,125 @@ header {
...
+    width: 80%;
+    margin-left: 10%;
+    max-width: 1200px;
+  }
+
+  .section-intro {
+    padding-left: 15%;
+    padding-right: 15%;
+    background-size: cover;
+  }
```
Comment (AbelG101) [inline assets/css/style.css]: - [x] Ouch! this is close but please make sure the background image of the landing page is aligned properly as shown in the Figma design:
- Your design:
  image
- Figma design:
  image

File: index.html [hunk 2]
```diff
@@ -273,20 +284,22 @@ <h2 class="connect">
...
+            <div class="contact-form">
+                <form action="https://formspree.io/f/xpzebrvw" method="POST">
+                    <input type="text" name="name" maxlength="30" placeholder="Lorem Ipsum" required>
+                    <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                    <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here..." required></textarea>
+                    <button class="project-link" type="submit">Get in touch</button>
+                </form>
+            </div>
         </div>
     </section>
```
Comment (AbelG101) [inline index.html:304]: - [x] As mentioned previously pelase make sure the background image for the "Contact me" section is aligned properly as shown in the Figma design: 
- Your design:
  image
- Figma design:
  image

- [x] Also please make sure to remove any section descriptions from your readme as they're only there to guide the author on what to place on each section: 👇
    image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Aqurds 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aamirkhan2478 — APPROVED** [review]
Hi @Aqurds , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**